services/proctoring-service/.env

```
@@ -1,22 +1,9 @@
-# Application Port
-PORT=8082
-
-# PostgreSQL Database Configuration
-DB_HOST=localhost
-DB_PORT=5433
 DB_USER=postgres
 DB_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)